File: 프로그래머스/src/프로그래머스_02252022/가운데_글자_가져오기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s)
{
    string answer = "";
    int mid = s.size() / 2 - 1;
    if (s.size() == 1)
        return s;
    if (s.size() % 2 == 0)
        answer = s.substr(mid, 2);
    else
        answer = s.at(mid + 1);

    return answer;
}

File: 2026/Level1/가운데글자가져오기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12903?language=python3
def solution(s):
    middle = int(len(s)/ 2) -1
    # even else odd
    answer = s[middle:middle+2] if len(s)%2==0 else s[middle+1] 
    return answer